modules/calendar/campus.py

```diff
@@ -0,0 +1,4 @@
+class Campus:
```

Comment:
To be moved to `/utils/campus/campus.py`.

`get_campuses` should be a public static method of this class